driverless/src/main/java/com/upgrad/hirewheels/controller/AuthenticationController.java

```diff
@@ -36,13 +49,27 @@ public ResponseEntity userLogin(@RequestBody LoginDTO loginDTO){
             try {
                 userValidator.validateuserLogin(loginDTO);
                 User userDetail = userService.getUserDetails(loginDTO);
+                //JWT Starts
+                //JWT Changed to Email
+                String refreshToken = jwtTokenProvider.createRefreshToken(userDetail.getEmail());
+                String token = jwtTokenProvider.createToken(userDetail.getEmail());
+                String oldToken = userDetail.getJwtToken();
+                userDetail.setJwtToken(token);
+                userDetail.setRefreshToken(refreshToken);
+                userServiceImpl.addToken(token);
```

Comment:
If we are planning to store it in db, can do all these operations in a single call to minimize the db calls